scripts/validate-changelog.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
```

Comment:
Part of the CI/CD process to ensure that the changelog for a specific package is valid before allowing a release or merge